### PR TITLE
compile src via babel

### PR DESCRIPTION
**Pull request by sabeurthabti**
a demo project: https://github.com/sabeurthabti/preact-fluid-test-area

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,27 +1,30 @@
...
-    "prettier --write src/**/*.js",
-    "prettier-check":
-    "prettier --list-different src/**/*.js",
+    "prettier": "prettier --write src/**/*.js",
+    "prettier-check": "prettier --list-different src/**/*.js",
     "publish-docs": "npm run styleguide-build && npm run gh-pages",
-    "publish-patch": "npm version patch && npm publish",
-    "publish-minor": "npm version minor && npm publish",
-    "publish-major": "npm version major && npm publish"
+    "publish-patch": "npm run build && npm version patch && npm publish",
```
Comment (sabeurthabti) [inline package.json]: there is probably a nicer way to do this

File: package.json [hunk 2]
```diff
@@ -1,27 +1,30 @@
...
+    "publish-minor": "npm run build && npm version minor && npm publish",
+    "publish-major": "npm run build && npm version major && npm publish"
   },
   "lint-staged": {
-    "*.{js}": ["prettier --write src/**/*.js", "git add"]
+    "*.{js}": [
+      "prettier --write",
```
Comment (sabeurthabti) [inline package.json]: with lint-staged, whatever is staged on git will get appended to the command.
`prettier --write [GIT_STAGE_FILES]`

File: package.json [hunk 3]
```diff
@@ -1,27 +1,30 @@
 {
-  "name": "preact-fluid",
+  "name": "preact-fluid-test",
```
Comment (ajainvivek) [inline package.json]: Change the name to preact-fluid ;)

File: package.json (hunk 1 above)
Comment (ajainvivek) [reply package.json]: looks good

File: package.json (hunk 2 above)
Comment (ajainvivek) [reply package.json]: that's right

**Comment by ajainvivek** [comment]
@sabeurthabti Great work mate 👍

**Comment by sabeurthabti** [comment]
@ajainvivek update with the correct name

**Pull request merged**